File: src/entity/category.ts
export { Category };

class Category {
	public id?: number | null;
	public name: string;
	public sequence?: number | null;
	public account_ids: number[] = [];

	constructor(id: number | null, name: string) {
		this.id = id;
		this.name = name;
	}
}

File: src-tauri/src/entity/category.rs
use serde::{Serialize, Deserialize};

#[derive(Debug, Serialize, Deserialize)]
pub(crate) struct Category {
    pub(crate) id: Option<i32>,
    pub(crate) name: String,
    pub(crate) sequence: Option<i32>,
    pub(crate) last_update_time: Option<String>,
    // 外部字段
    pub(crate) account_ids: Option<Vec<i32>>,
}